### PR TITLE
test: set up test directory

### PR DESCRIPTION
**Pull request by VictorChao996**
### Description:

- Notion link: https://www.notion.so/SLD-7-Test-set-up-test-folder-344f809b921749a2b7788962f89628ea?pvs=4
- Set up test folder & README.md file for backend API local testing.

### Changes:

- add healthcheck.test.js for local API testing.
- add README.md file

### Screenshots (optional)
image

### CODE REVIEW

**Comment by VictorChao996** [comment]
想找 Michael review 結果發現他還不是 contirbutor 😂

File: test/healthcheck.test.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import require from "supertest";
+import app from "../server/app.js";
+
+describe(`healthcheck`, () => {
+    it("should return 200 OK status", async () => {
+        const res = await require(app).get("/api/v1/healthcheck").expect(200);
+
+        // expect(res.body).toHaveProperty("status", "ok");
+        expect(res.body).toHaveProperty("sf");
```
Comment (YuakaTL) [inline test/healthcheck.test.js:9]: Should we swap the comments of these two lines?

Comment (VictorChao996) [reply test/healthcheck.test.js:9]: 我的鍋 昨天示範完之後忘記改

**Review by 5-michelle — APPROVED** [review]
LGTM!

**Pull request merged**